dask/utils.py

```diff
@@ -18,7 +18,7 @@
 from numbers import Integral, Number
 from operator import add
 from threading import Lock
-from typing import ClassVar, TypeVar
+from typing import Callable, ClassVar, TypeVar
```

Comment:
this is deprecated; please import the same from collections.abc instead

Reply:
Sorry if I wasn't clear: Callable itself is fine, but it should be

``from collections.abc import Callable``
instead of
``from typing import Callable``